ansible/vars/topo_t2.yml

```diff
@@ -38,19 +38,19 @@ topology:
     ARISTA13T3:
       vlans:
         - 0.12@12
-        - 0.13@13
       vm_offset: 6
     ARISTA15T3:
       vlans:
+        - 0.13@13
         - 0.14@14
-        - 0.15@15
       vm_offset: 7
     ARISTA17T3:
       vlans:
-        - 0.16@16
+        - 0.15@15
       vm_offset: 8
     ARISTA18T3:
       vlans:
+        - 0.16@17
```

Comment:
the ptf port index is the  same should it 
` 0.16@17
 0.17@17
`

Reply:
thanks @arlakshm . Fixed. Looks like this ptf index is not used in `topo_fact` so i did not saw any issue.